helpers/setup.py

```diff
@@ -131,3 +189,32 @@ def update_hosts(cls, config):
             return_value = os.system("sudo mv /etc/hosts /etc/hosts.old && sudo mv /tmp/etchosts /etc/hosts")
             if return_value != 0:
                 sys.exit(1)
+
+    @staticmethod
+    def validate_already_run():
```

Comment:
sure, as you suggested :wink:, let's add a docstring—maybe something like: "make sure that `kobo-docker` has already been cloned from the versioning repository by Git"?